File: Coding Club Beginner Projects/Lesson4/grades.cpp
#include <iostream>

int main()
{
    int amountOfGrades = 0;
    std::cout << "Hello! Enter in the amount of grades you want to see the letter for.";
    std::cin >> amountOfGrades;

    char letterGrades[amountOfGrades];

    double grade = 0;

    for (int i = 0; i < amountOfGrades; i++)
    {
        std::cout << "Enter in grade number " << i + 1 << ": ";
        std::cin >> grade;

        if (grade >= 90 && grade <= 100)
        {
            letterGrades[i] = 'A';
        }
        else if (grade >= 80 && grade <= 89)
        {
            letterGrades[i] = 'B';
        }
        else if (grade >= 70 && grade <= 79)
        {
            letterGrades[i] = 'C';
        }
        else if (grade >= 60 && grade <= 69)
        {
            letterGrades[i] = 'D';
        }
        else if (grade >= 0 && grade <= 59)
        {
            letterGrades[i] = 'F';
        }
        else
        {
            std::cout << "Invalid grade entered... Please enter in a valid grade.\n";
            i--;
        }
    }

    for (int i = 0; i < amountOfGrades; i++)
    {
        std::cout << "Here are the grades:\n";
        std::cout << letterGrades[i] << '\n';
    }
}

File: Coding Club Beginner Projects/Lesson4/grades.py
amount_of_grades = int(input("Hello! Enter in the amount of grades you want to see the letter for."))

letter_grades = []

for i in range(amount_of_grades):
    print("Enter in grade number", i + 1, ": ", end='')
    grade = float(input(""))
    if grade >= 90 and grade <= 100:
        letter_grades.append('A')
    elif grade >= 80 and grade <= 89:
        letter_grades.append('B')
    elif grade >= 70 and grade <= 79:
        letter_grades.append('C')
    elif grade >= 60 and grade <= 69:
        letter_grades.append('D')
    elif grade >= 0 and grade <= 59:
        letter_grades.append('F')
    else:
        print("Invalid grade entered... Please enter in a valid grade.")
        i -= 1

print("Here are the grade:")
for letter_grade in letter_grades:
    print(letter_grade)